src/main/java/org/opentripplanner/street/model/StreetLimitationParametersServiceModule.java

```diff
@@ -0,0 +1,21 @@
+package org.opentripplanner.street.model;
+
+import dagger.Module;
+import dagger.Provides;
+import jakarta.inject.Singleton;
+
+/**
+ * The service is used during application serve phase, not loading, so we need to provide
+ * a module for the service without the repository, which is injected from the loading phase.
+ */
+@Module
+public class StreetLimitationParametersServiceModule {
+
+  @Provides
+  @Singleton
+  public StreetLimitationParametersService provideEmissionsService(
```

Comment:
I think you want to rename the method. :)

Reply:
Perhaps :)